apps/nodejs/extension/src/ui/Transaction/RecipientAutocomplete.tsx

```diff
@@ -195,10 +195,11 @@ export default function RecipientAutocomplete({
     return isValidAddress(value, protocol);
   };
 
-  const invalidValue =
-    !!inputValue &&
+  const invalidValue = useMemo(() => {
```

Comment:
Why use useMemo here?

Reply:
Just a minor optimization I did while debugging. Granted this calculation is minor, probably not needed.